octoprint_mrbeam/translations/de/LC_MESSAGES/messages.po

```diff
@@ -2266,84 +2266,92 @@ msgstr ""
 "Dies kann bei empfindlichem Material zu Brandflecken auf der linken und "
 "rechten Seite der Gravur führen."
 
-#: octoprint_mrbeam/templates/conversion_dialog.jinja2:448
+#: octoprint_mrbeam/templates/conversion_dialog.jinja2:449
 msgid "Prevent unwanted burn marks"
 msgstr "Unerwünschte Brandflecken verhindern"
 
-#: octoprint_mrbeam/templates/conversion_dialog.jinja2:455
+#: octoprint_mrbeam/templates/conversion_dialog.jinja2:450
+msgid ""
+"This feature has been disabled as the burn marks issue has been resolved in "
+"our Software"
```

Comment:
It's not ending with dot here although it's added above and in german... 🤔

Reply:
good catch, I added the dot after the translation, will redo it